DelHel/CDelHel.cpp

```diff
@@ -209,6 +212,31 @@ bool CDelHel::OnCompileCommand(const char* sCommandLine)
 			this->SaveSettings();
 			this->CheckLoadedPlugins();
 
+			return true;
+		}
+		else if (args[1] == "config")
+		{
+			if (args[2] == "none")
+			{
+				this->LogMessage("No longer using custom config", "Config");
+				this->customConfigActive = false;
+				this->activeConfig = "";
+
+				return true;
+			}
+
+			auto config = this->configs.find(args[2]);

```

Comment:
```suggestion
			auto config = this->configs.find(to_upper(args[2]));

```
I would suggest converting the config keys as well as user input to uppercase (or lowercase, whichever you prefer, but `to_upper` is already a defined helper) just to make selecting a bit easier without having to match config casing (so `f1` would work same as `F1`).